src/points/cardDefinitions.tsx

```diff
@@ -40,3 +49,43 @@ export default function useCardDefinitions(
     },
   }
 }
+
+export function useGetHistoryDefinition(): (history: ClaimHistory) => HistoryCardMetadata {
+  const { t } = useTranslation()
+  const tokensById = useSelector((state) =>
+    tokensByIdSelector(state, getSupportedNetworkIdsForSwap())
+  )
+
+  return (history: ClaimHistory) => {
+    switch (history.activityId) {
+      case 'create-wallet': {
+        return {
+          icon: <Celebration color={colors.successDark} />,
+          title: t('points.history.cards.createWallet.title'),
+          subtitle: t('points.history.cards.createWallet.subtitle'),
+          pointsAmount: history.pointsAmount,
+        }
+      }
+      case 'swap': {
+        const fromToken = tokensById[history.metadata.from]
+        const toToken = tokensById[history.metadata.to]
+        if (!fromToken || !toToken) {
+          throw new Error(`Cannot find tokens ${history.metadata.from} or ${history.metadata.to}`)
```

Comment:
rather than throwing an error, can we just silently omit this entry? or we can do a Logger.error so that it gets sent to Sentry, but idk if it's worth the extra error handling to do that here?